src/.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Correct me if I'm wrong, but the `.prettierrc` file should exist in the project root, not under `src/`?

https://prettier.io/docs/en/configuration.html

Reply:
Yes. It should be in the root.

Reply:
It should be. that was a mistake.